external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -986,7 +1003,7 @@ pub fn get_symbols(
     let mut parsed_ast = None;
     let mut typed_ast = None;
     let mut diagnostics = None;
-    build_plan.compile_with_driver(&mut std::io::sink(), |compiler| {
+    build_plan.compile_with_driver(Some(Box::new(vfs)), &mut std::io::sink(), |compiler| {
         // extract expansion AST
```

Comment:
That is what I had here initially, but since I have to use it before/after `compiler_with_driver` call to retrieve file sources for the symbolicator to use, I could not figure out how to avoid copying/cloning of `vfs`. In particular, I can't make the closure into the `move` one as we have things that we need to copy out of it. It's likely a gap in my Rust skills, so suggestions welcome!

Reply:
Not sure that I follow. I don't see it being used after `compile_with_driver`. And before shouldn't matter ?

Reply:
This is indeed the direction I went for.